### PR TITLE
document access log location

### PR DESCRIPTION
**Pull request by bparees**
documents the issue in https://bugzilla.redhat.com/show_bug.cgi?id=1231862

@thoraxe @danmcp @adellape ptal

### CODE REVIEW

File: using_images/s2i_images/php.adoc [hunk 1]
```diff
@@ -143,3 +143,6 @@ If the `*DocumentRoot*` of the application is nested in the source directory
 the default Apache behavior and specify how application requests should be
 handled. The *_.htaccess_* file must be located at the root of the application
 source.
+
+== Logs
+This image logs primarily to standard out and as such the logs can be viewed via the `*oc logs*` command.  Access logs are stored in *_/tmp/access_log_* which can be viewed use `*oc exec*` to access the container.
```
Comment (adellape) [inline using_images/s2i_images/php.adoc]: s/use/using/

Maybe also sync up on "via" or "using" in the two sentences (would suggest "using").

**Comment by adellape** [comment]
In absence of example commands, inline links to the following seem appropriate in the respective spots:

https://docs.openshift.org/latest/cli_reference/basic_cli_operations.html#troubleshooting-and-debugging-cli-operations

https://docs.openshift.org/latest/dev_guide/executing_remote_commands.html

And 1 comment inline, otherwise LGTM.

**Comment by bparees** [comment]
@adellape fixed, thanks

**Comment by adellape** [comment]
:+1:

**Comment by bparees** [comment]
@adellape merge?

**Comment by adellape** [comment]
@bparees Was seeing if anyone else had feedback. But yea, merge! :oncoming_automobile:

**Pull request merged**